src/ts2fable.fs

```diff
@@ -904,6 +905,29 @@ let fixThis(md: FsModule): FsModule =
 
     { md with Types = md.Types |> List.map (fixType fix) }
 
+let fixOpens(f: FsFile): FsFile =
+
+    let isBrowser (name: string) =
+        name.StartsWith "HTML"
```

Comment:
It would be better to see if it is a type that derives from `HTMLElement`, but let's see how far this gets us.